components/formats-gpl/src/loci/formats/in/ZeissCZIReader.java

```diff
@@ -88,6 +89,11 @@
 
   // -- Constants --
 
+  private static final String ALLOW_AUTOSTITCHING_KEY = "zeissczi.allow.autostitch";
```

Comment:
I was mostly modelling the names on https://github.com/openmicroscopy/bioformats/blob/develop/components/formats-api/src/loci/formats/in/DefaultMetadataOptions.java#L44.  I'm not especially attached to them, though, so happy to change if there is a better suggestion.